commands/team_e2e_test.go

```diff
@@ -51,3 +52,48 @@ func (s *MmctlE2ETestSuite) TestRenameTeamCmdF() {
 		s.Equal("Cannot rename team '"+s.th.BasicTeam.Name+"', error : : You do not have the appropriate permissions., ", err.Error())
 	})
 }
+
+func (s *MmctlE2ETestSuite) TestModifyTeamsCmdF() {
+	s.SetupTestHelper().InitBasic()
+	s.RunForSystemAdminAndLocal("system & local accounts can set a team to private", func(c client.Client) {
+		printer.Clean()
+		teamID := s.th.BasicTeam.Id
+		cmd := &cobra.Command{}
+		cmd.Flags().Bool("private", true, "")
+		err := modifyTeamsCmdF(c, cmd, []string{teamID})
+		s.Require().NoError(err)
+
+		t, appErr := s.th.App.GetTeam(teamID)
+		s.Require().Nil(appErr)
+		s.Require().Equal(model.TEAM_INVITE, t.Type)
+
+		// teardown
+		appErr = s.th.App.UpdateTeamPrivacy(teamID, model.TEAM_OPEN, true)
+		s.Require().Nil(appErr)
+	})
+	s.Run("user that creates the team can't set team's privacy due to permissions", func() {
+		printer.Clean()
+		teamID := s.th.BasicTeam.Id
+		cmd := &cobra.Command{}
+		cmd.Flags().Bool("private", true, "")
+		s.th.LoginBasic()
```

Comment:
Login is done as part of the `s.SetupTestHelper().InitBasic()`, so unless you logout it should not be necessary to log back in